service/service.go

```diff
@@ -53,6 +53,9 @@ type Service struct {
 	// a Git host.
 	Repository string `hash:"name:7"`
 
+	// Source is the hash id of service's source code on IPFS.
+	Source string
```

Comment:
```suggestion
	Source string `hash:"-"`
```
Source should be omitted from hash calculation until the cli can generate fully deterministic archive and get the same ifps hash all the time

Reply:
done